<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1550</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>770</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ffffff</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1550</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#9b9b9b</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>571</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1 Display Bold</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-left:150px;
color:#ffffff</string>
      </property>
      <property name="text">
       <string>Staff Search Tool!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/Thiết_kế_chưa_có_tên__2_-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1350</x>
        <y>5</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/UICON/img/365557112_256733483966404_2319603164221613727_n-fotor-20230814142752.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>1410</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Administrator</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#9b9b9b</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/Thiết kế chưa có tên (1).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>180</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/contact (1).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="calc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/UICON/accounting.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="insert">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/UICON/add-user.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="home">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/UICON/customer (1).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="search">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Hello/Downloads/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>1350</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Employee's ID</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date of birth</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gender</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email address</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Phone numbers</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Position</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Basic salary</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bank account numbers</string>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>1450</x>
       <y>0</y>
       <width>100</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#9b9b9b</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>550</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF;
border:1px solid black;
padding-left:20px;


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Add_2">
     <property name="geometry">
      <rect>
       <x>605</x>
       <y>780</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>19</pointsize>
       <italic>true</italic>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#ffffff;
border:1px solid black;
border-radius:20px</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Pos">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>410</y>
       <width>550</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF;
border:1px solid black;
padding-left:20px

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Gender">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>480</y>
       <width>550</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF;
padding-left:20px;
border: 1px solid black

</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nam</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nữ</string>
      </property>
     </item>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>frame_3</zorder>
    <zorder>frame_4</zorder>
    <zorder>tableWidget</zorder>
    <zorder>Name</zorder>
    <zorder>Add_2</zorder>
    <zorder>Pos</zorder>
    <zorder>Gender</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1550</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
